src/App/adapter/translate.js

```diff
@@ -1,18 +1,30 @@
 import { createInclude, createWhere } from './lib/create'
 import { stripEmptyKeys } from './lib/helpers'
-import { mergeState, parseState } from './lib/state'
+import { createPagination, mergeState, parseState } from './lib/state'
+
+const DEFAULT_CONFIG = {
+  endpoint: 'documents',
+  include: [],
+  pageSize: 5, // `false` to disable limit/pagination
+  page: 1,
+  order: undefined, // e.g.`['foo ASC', 'bar DESC']`
```

Comment:
Declaring `order` explicitly here for documentation purposes. If we ever switch to TypeScript, this won't be needed.